diff --git a/node_modules/unimport/dist/shared/unimport.d165baa5.cjs b/node_modules/unimport/dist/shared/unimport.d165baa5.cjs
index e2ca34d..1248928 100644
--- a/node_modules/unimport/dist/shared/unimport.d165baa5.cjs
+++ b/node_modules/unimport/dist/shared/unimport.d165baa5.cjs
@@ -1042,15 +1042,13 @@ async function scanExports(filepath, includeTypes, seen = /* @__PURE__ */ new Se
         } else {
           const subfile = exp.specifier;
           let subfilepath = pathe.resolve(pathe.dirname(filepath), subfile);
-          if (!pathe.extname(subfilepath)) {
-            for (const ext of FileExtensionLookup) {
-              if (node_fs.existsSync(`${subfilepath}${ext}`)) {
-                subfilepath = `${subfilepath}${ext}`;
-                break;
-              } else if (node_fs.existsSync(`${subfilepath}/index${ext}`)) {
-                subfilepath = `${subfilepath}/index${ext}`;
-                break;
-              }
+          for (const ext of FileExtensionLookup) {
+            if (node_fs.existsSync(`${subfilepath}${ext}`)) {
+              subfilepath = `${subfilepath}${ext}`;
+              break;
+            } else if (node_fs.existsSync(`${subfilepath}/index${ext}`)) {
+              subfilepath = `${subfilepath}/index${ext}`;
+              break;
             }
           }
           if (!node_fs.existsSync(subfilepath)) {
diff --git a/node_modules/unimport/dist/shared/unimport.e70d488c.mjs b/node_modules/unimport/dist/shared/unimport.e70d488c.mjs
index a985a0a..c439a3b 100644
--- a/node_modules/unimport/dist/shared/unimport.e70d488c.mjs
+++ b/node_modules/unimport/dist/shared/unimport.e70d488c.mjs
@@ -1034,15 +1034,13 @@ async function scanExports(filepath, includeTypes, seen = /* @__PURE__ */ new Se
         } else {
           const subfile = exp.specifier;
           let subfilepath = resolve(dirname(filepath), subfile);
-          if (!extname(subfilepath)) {
-            for (const ext of FileExtensionLookup) {
-              if (existsSync(`${subfilepath}${ext}`)) {
-                subfilepath = `${subfilepath}${ext}`;
-                break;
-              } else if (existsSync(`${subfilepath}/index${ext}`)) {
-                subfilepath = `${subfilepath}/index${ext}`;
-                break;
-              }
+          for (const ext of FileExtensionLookup) {
+            if (existsSync(`${subfilepath}${ext}`)) {
+              subfilepath = `${subfilepath}${ext}`;
+              break;
+            } else if (existsSync(`${subfilepath}/index${ext}`)) {
+              subfilepath = `${subfilepath}/index${ext}`;
+              break;
             }
           }
           if (!existsSync(subfilepath)) {
